src/components/DataSession/OperationWizard.vue

```diff
@@ -176,21 +172,13 @@ const handleThumbnailClick = (item) => {
             v-else-if="inputDescription.type == 'file'"
             class="images-container"
           >
-            <v-col
-              v-for="image in images"
-              :key="image.basename"
-              :cols="calculateColumnSpan(images.length)"
+            <image-grid 
+              :data="data"
+              :column-span="calculateColumnSpan(data.input_data.length, 5)"
+              :selected-images="selectedDataSessionImages"
               class="wizard-images"
-            >
-              <v-img
-                :src="image.url"
-                :alt="image.basename"
-                cover
-                aspect-ratio="1"
-                :class="{ 'selected-image': selectedDataSessionImages.some(selectedImage => selectedImage.basename === image.basename) }"
-                @click="handleThumbnailClick(image)"
-              />
-            </v-col>
+              @image-clicked="handleThumbnailClick"
```

Comment:
added this emit because imagegrid is a whole row so if we clicked on an image, the whole row would get a green border